### DISCUSSION TITLE
Route group resources: second non-transparent parameter in the documentation

### DISCUSSION THREAD

**Discussion by marcelosb** [laravel/framework#42781] [Q&A]
in https://laravel.com/docs/9.x/controllers#resource-controllers 

You may even register many resource controllers at once by passing an array to the resources method:
```
Route::resources([
    'photos' => PhotoController::class,
    'posts' => PostController::class,
]);
```

However, there is a second parameter that is not clear
In documentation API: **static void resources(array $resources, array $options = [])**

What is the options parameter for?

it works?
```
Route::resources([
    'photos' => PhotoController::class,
    'posts' => PostController::class,
], ['except' => 'destroy']);
```

**Comment by juse-less** [comment]
I clicked my way through the DocBlocks, and eventually ended up in [`PendingResourceRegistration`](https://github.com/laravel/framework/blob/ccefc764871fe263dd20f8f7108c18180f78241b/src/Illuminate/Routing/PendingResourceRegistration.php).
You can see [options being set from its constructor](https://github.com/laravel/framework/blob/ccefc764871fe263dd20f8f7108c18180f78241b/src/Illuminate/Routing/PendingResourceRegistration.php).

If you then look a few lines below, you can see that the various methods, like [`PendingResourceRegistration::except()`](https://github.com/laravel/framework/blob/ccefc764871fe263dd20f8f7108c18180f78241b/src/Illuminate/Routing/PendingResourceRegistration.php), indeed sets that option.
So your assumption in your example seems to be correct. 🙂
_Possibly_ that you may need to enclose `'destroy'` in an array - `['except' => ['destroy']]`.

I guess one could say that it's used to set the typical options we normally call on 'singular routes', like `Route::resource()->except()`, but for when you do it on 'plural routes', like `Route::resources()`.

**Reply by marcelosb** [reply]
Hello,

True, that's what I was thinking.

But, I think those responsible for the documentation could add this information from the options.

**Thanks.**
